### ISSUE TITLE
Server blocked

### ISSUE THREAD

**Issue by brettstack** [Armax/Pokemon-GO-node-api#188]
Welp, Niantic has started blocking servers. Are there any proxy services that we can optionally configure to work with this lib?

**Comment by chriseckman** [comment]
I think mine has been blocked as well. Also, Google shot down an update to
my app on the Play Store because of likely using Niantic's severs.

**Comment by coolaj86** [comment]
The reason that my company, Daplie, is interested in the whole Pokemon GO pokemaps scene is for exactly this reason: it's one of the first widespread applications that requires a home server.

We're working on getting two of our products (Daplie Domains - a better dynamic dns - and Daplie IP - a reverse vpn) integrated into our [Pokémap server](https://github.com/Daplie/node-pokemap) (and would love some help on the open source bits).

Here are some tips:

* Use your home IP address, not heroku, digital ocean, etc
* (if you don't have a public IP you can use OpenVPN an an inbound-only mode to use an external IP for web access, but the home ip for all api calls)
* Don't allow multiple users in distant locations from the same IP
* Don't allow more than a dozen users to login anyway
* Don't move the player around the map at unrealistic speeds (heartbeats act as movement)
* Basically: behave like a real client - as if you were just someone using wifi in the airport - in all respects that you can.

**Comment by brettstack** [comment]
That doesn't help those running on AWS and co.

**Comment by jdpnielsen** [comment]
Not much we can do from our side. Please take this convo to the [Discord](https://discord.gg/W9CZTgn) server or [pokemongodev](https://www.reddit.com/r/pokemongodev/) subreddit.

**Comment by jiaming-jx** [comment]
Have you guys figured out any good ways to deal with this problem?

**Comment by rafaelcorreiapoli** [comment]
Requests from heroku are not working either... is this related to this?
edit:
There is a proxy Addon on heroku called Fixie
https://devcenter.heroku.com/articles/fixie
But we need to pass some parameters to the request methods made by this lib in order to use the proxy
https://devcenter.heroku.com/articles/fixie#using-with-node
